frontend/src/data/clusters/reducer.js

```diff
@@ -33,7 +33,7 @@ export function reducer(state, action) {
         case CLUSTER_ACTION_TYPES.SET_CURRENT_CLUSTER:
             return {
                 ...state,
-                currentCluster: action.currentCluster,
+                currentCluster: action.clusterSlug,
```

Comment:
Have we changed the action creator to match this? Not sure if I'm just being blind but I can't see it?

Reply:
I think this was the error that was messing up the Cluster selection, we were passing down clusterSlug but expecting currentCluster.

Reply:
Ah yeah - looks like the clusters action creators are inconsistent:

https://github.com/EDITD/ElasticSurgery/blob/master/frontend/src/data/clusters/actions.js#L11 calls it `clusterSlug`, whilst https://github.com/EDITD/ElasticSurgery/blob/master/frontend/src/data/clusters/actions.js#L33 calls it `currentCluster`. That definitely needs fixed!